pkg/operator/controller/controller_dns_configmap_test.go

```diff
@@ -644,6 +772,35 @@ func TestDesiredDNSConfigmapUpstreamResolvers(t *testing.T) {
 	}
 }
 
+func TestCannotConfigureForwardPluginTransportTLSWithoutServerName(t *testing.T) {
```

Comment:
Since you are testing `errTransportTLSConfiguredWithoutServerName`, is it also fair to consider testing for `errInvalidNetworkUpstream`?

Reply:
Ah great, looks good